### DISCUSSION TITLE
R - Pengujian perbedaan CWB dan WM pada formasi 3412, apakah sudah cukup ilmiah?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#24] [Q&A] [answered]
Permisi, ini saya coba menguji formasi 3412 apakah signifikan jika sayapnya dibedakan antara menggunakan wide midfielder atau complete wing back. 

Saya putuskan mengunakan bootstrap dan permutation untuk hipotesis testnya karena relatif lebih mudah dipahami. 
Apakah ini sudah cukup ilmiah? Apakah ada rekomendasi tes lainnya? Tadinya mau menggunakan chi square sesuai arahan Pak Muh Risman, namun masih ragu karena belum paham konsepnya, jadinya pakai perbedaan median saja. 

https://www.kaggle.com/code/husadahertzsprung/side-positional-role-3412-formation

-

_oiya referensi saya menjalankan tes ini adalah videonya professor Marin di youtub_e

**Comment by muhrisman** [comment, marked as answer]
Dear bapak @Hertzspr

Bapak telah melakukan analisa pengujian dua sampel dengan baik. Fokus dari pengujian tersebut adalah apakah plan A berbeda secara signifikan dengan plan B.

Terkait hasil analisa yang bapak lakukan, ada beberapa feedback yang bisa saya berikan.

P-value 0.05 nilai threshold yang umum digunakan namun bukan berarti nilai ini adalah batas mutlak penentuan signifikansi suatu kriteria. P-value bisa disesuaikan menyesuaikan kebutuhan. Namun, semakin kecil nilai p-value berarti semakin menunjukan efek signifikan diantara dua kelompok. Mengacu pada ini, dapat dibilang pengujian yang bapak lakukan sudah relatif signigikan, Plan A berbeda secara signifikan dengan plan B.
Untuk meyakinkan hasil analisa, bapak bisa coba dengan membandingkannya menggunakan mean. Tentunya mean yang dibandingkan harus memperhatikan aspek outlier yang ada di dalamnya.
Terdapat bagian visualisasi yang butuh perhatian ekstra untuk dimengerti, bapak bisa tambahkan keterangan visualisasi agar user dapat mengerti penjelasan plot tersebut lebih lanjut. Misal pada plot kc_sum_by_role
Confidence interval yang berada diantara nilai minus dan positif akan berisi kemungkinan nilai 0. Nilai 0 yang berada pada confidence interval dapat berarti tidak terdapat efek yang signifikan pada pengujian. Hal ini perlu diperhatikan dengan 2 alasan: 1) pengujian memang menghasilkan tidak terdapatnya perbedaan yang signifikan pada kedua group, 2) terdapat alur pengujian yang kurang tepat. Terkait dengan alur pengujian yang kurang tepat, bisa dikembalikan lagi pada metrik yang dipilih.
Hasil uji signifikansi dan confidence interval berhubungan satu sama lain. Jika dalam pengujian signifikansi menunjukan hasil tidak signifikan, maka ada kemungkinan confidence interval akan menghasilkan rentang nilai 0 di dalamnya.
Demikian feedback untuk latihan yang bapak kerjakan,
semoga bermanfaat :)

**Reply by Hertzspr** [reply]
Terima kasih Pak.
